nginx.conf

```diff
@@ -212,7 +212,6 @@ http {
                 dev-mikkeli.digitransit.fi mikkeli.digitransit.fi
                 dev-oulu.digitransit.fi oulu.digitransit.fi
                 dev-turku.digitransit.fi turku.digitransit.fi reittiopas.foli.fi
-                dev-tampere.digitransit.fi tampere.digitransit.fi repa.tampere.fi reittiopas.tampere.fi
```

Comment:
Should we keep one of the domains for old ui/otp1 so we can easily test if there are differences vs old ui/otp1 once we have gone to production with the new setup?